tests/e2e/cucumber/features/smoke/uploadResumable.ocis.feature

```diff
@@ -15,14 +15,12 @@ Feature: Upload large resources
       | largefile.txt |
     And "Alice" pauses the file upload
     And "Alice" cancels the file upload
-    # issue: https://github.com/owncloud/web/issues/9080
-    # Todo: uncomment when the issue is fixed
-    # And "Alice" starts uploading the following large resources from the temp upload directory
-    #   | resource      |
-    #   | largefile.txt |
-    # And "Alice" pauses the file upload
-    # And "Alice" resumes the file upload
-    # Then following resources should be displayed in the files list for user "Alice"
-    #   | resource      |
-    #   | largefile.txt |
+     And "Alice" starts uploading the following large resources from the temp upload directory
+       | resource      |
+       | largefile.txt |
+     And "Alice" pauses the file upload
+     And "Alice" resumes the file upload
+     Then following resources should be displayed in the files list for user "Alice"
+       | resource      |
+       | largefile.txt |
```

Comment:
```suggestion
   And "Alice" starts uploading the following large resources from the temp upload directory
     | resource      |
     | largefile.txt |
   And "Alice" pauses the file upload
   And "Alice" resumes the file upload
   Then following resources should be displayed in the files list for user "Alice"
     | resource      |
     | largefile.txt |
```

Reply:
The original PR also had this small indent problem.
@SagarGi you could make a separate PR to fixup any indent problems in feature files (and do 2 PRs, to master and stable)